server/build.go

```diff
@@ -145,6 +149,26 @@ func (task *BuildTask) Build() (esm *ESM, err error) {
 }
 
 func (task *BuildTask) build() (esm *ESM, err error) {
+	if task.Target == "raw" {
+		filePath := path.Join(task.wd, "node_modules", task.Pkg.Name, task.Pkg.Submodule)
```

Comment:
seems here missed the package version, use `task.Pkg.String()` instead

Reply:
There is the npm cache dir, I think this doesn't need the version

Reply:
i see